src/Raven.Server/Documents/Sharding/Commands/StartManualReshardingCommand.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Operations;
+using Raven.Client.Http;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Sharding.Commands
+{
+    public sealed class StartManualReshardingOperation : IMaintenanceOperation
```

Comment:
why do we need this?

Reply:
only for testing.. moved to `RavenDB_22200.cs`